x-pack/legacy/plugins/index_management/public/sections/index_list/components/index_actions_context_menu/index_actions_context_menu.js

```diff
@@ -385,40 +387,42 @@ export class IndexActionsContextMenu extends Component {
     const selectedIndexCount = indexNames.length;
 
     const standardIndexModalBody = (
-      <div>
+      <Fragment>
         <p>
           <FormattedMessage
             id="xpack.idxMgmt.indexActionsMenu.deleteIndex.deleteDescription"
             defaultMessage="You are about to delete {selectedIndexCount, plural, one {this index} other {these indices} }:"
             values={{ selectedIndexCount }}
           />
         </p>
+
         <ul>
           {indexNames.map(indexName => (
             <li key={indexName}>
               {indexName}
             </li>
           ))}
         </ul>
+
         <p>
           <FormattedMessage
             id="xpack.idxMgmt.indexActionsMenu.deleteIndex.deleteWarningDescription"
             defaultMessage="You can't recover a deleted index. Make sure you have appropriate backups."
           />
         </p>
-      </div>
+      </Fragment>
     );
 
     const systemIndexModalBody = (
-      <div>
-        <EuiSpacer size="l" />
```

Comment:
This spacer created too much space between title and body.

![image](https://user-images.githubusercontent.com/1238659/60385829-c7ac7d00-9a42-11e9-8cee-14290ec8a307.png)